main.go

```
@@ -267,6 +300,31 @@ func main() {
 		os.Exit(1)
 	}
 
+	var etcdMgrOpts etcdmanager.Options = etcdmanager.Options{
+		EtcdImage:              etcdImage,
+		EtcdBackupSidecarImage: etcdBackupSidecarImage,
+		SecretKeyRefName:       etcdBackupSecretName,
+		PostgresletNamespace:   postgresletNamespace,
+		PartitionID:            partitionID,
+		PSPName:                etcdPSPName,
+		PostgresletFullname:    postgresletFullname,
+	}
+	etcdMgr, err := etcdmanager.New(svcClusterConf, "external/svc-etcd.yaml", scheme, ctrl.Log.WithName("EtcdManager"), etcdMgrOpts)
+	if err != nil {
+		setupLog.Error(err, "unable to create `EtcdManager`")
+		os.Exit(1)
+	}
+	if deployEtcd {
+		if err = etcdMgr.InstallOrUpdateEtcd(); err != nil {
+			setupLog.Error(err, "unable to deploy etcd")
+			os.Exit(1) // TODO should we really exit here?
```

remove TODO i prefer to fail early.